src/components/Clickable/Clickable.tsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
```

Comment:
how many of these errors did you have in this file? perhaps consider moving this `eslint-disable` to each relevant line (assuming there aren't too many)?